bundles/org.openhab.binding.automower/README.md

```diff
@@ -0,0 +1,80 @@
+# Automower Binding
+
+This binding communicates to the Husqvarna Automower Connect API in order to send commands and query the state of Husqvarna Automower robots.
+
+## Supported Things
+
+bridge: The bridge needs to be configured with credentials and an application key that allows communicating with the Automower Connect Api
+automower: A single Husqvarna Automower robot
+
+Basically all Husqvarna Automower models with "Automower Connect" support should be supported. It was tested only with a Husqvarna Automower 450X
+
+
+## Discovery
+
+Once the bridge is created and configured, registered automowers will be discovered automatically
```

Comment:
Exactly, the "automower" represents a single automower robot. The bridge supports communicating to multiple such automower robots. This should be in line with the rest apis / the husqvarna app where one registered user can have multiple robots.

Reply:
Ok, can you add a linebreak, since this formatting is a bit misleading:
![grafik](https://user-images.githubusercontent.com/1894841/87221315-7defb580-c36b-11ea-8c11-a9074088167d.png)

You could also add backticks to the Thing Type IDs (bridge, automower) to clarify these are the IDs.

Reply:
sure